src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs

```diff
@@ -84,7 +84,7 @@ public CdnjsCatalog(CdnjsProvider provider)
                     {
                         var completion = new CompletionItem
                         {
-                            DisplayText = version,
+                            DisplayText = $"{name}@{version}",
```

Comment:
>DisplayText = $"{name}@{version}", [](start = 28, length = 34)

Thinking of make the completion list for cdnjs and unpkg to be consistent.
The name@version will display the whole completion string while version only display the version part.

Reply:
Let's make unpkg match the old behavior here.  Only show either name or version (depending on context), and use name@version for the insertion text in both cases.

---
In reply to: [198361271](https://github.com/aspnet/LibraryManager/pull/258#discussion_r198361271) [](ancestors = 198361271)

Reply:
I have opened an issue to track it.

---
In reply to: [198665798](https://github.com/aspnet/LibraryManager/pull/258#discussion_r198665798) [](ancestors = 198665798,198361271)